Makefile

```diff
@@ -1,25 +1,183 @@
-NAME:=vrt
-LICENSE:="OSI Approved :: Apache Software License v2.0"
-AUTHOR:="Afiniate, Inc."
-ORGANIZATION:="afiniate"
-HOMEPAGE:="https://github.com/afiniate/vrt"
+# =============================================================================
+# Build Management Knobs
+# =============================================================================
 
-DEV_REPO:="git@github.com:afiniate/vrt.git"
-BUG_REPORTS:="https://github.com/afiniate/vrt/issues"
+PARALLEL_JOBS ?= 2
```

Comment:
In linux you can get the number of available processors with `nproc`. The problem is that there is no way to know whether that is what you really want, but maybe is a reasonable default.